src/components/ToggleButton.jsx

```diff
@@ -0,0 +1,27 @@
+import { array, string, func } from "prop-types";
+import styles from "./ToggleButton.module.css";
+
+export default function ToggleButton({ options, value, onSelect }) {
+  return (
+    <div className={styles["button-group"]}>
+      {options.map((option) => (
+        <button
+          type="button"
+          key={option.value}
+          className={`${styles.button} ${
+            value === option.value ? styles.selected : undefined
+          }`}
```

Comment:
```suggestion
          className={`${styles.button} ${
            value === option.value ? styles.selected : ''
          }`}
```
Leaving `undefined` here would result in the `className` string containing the literal text `undefined`. Using an empty string `('')` instead ensures the className remains clean.